scripts/recording_uploader/uploader/requirements.txt

```diff
@@ -0,0 +1 @@
+boto3==1.34.84
```

Comment:
What do you think about moving all of this outside of `/scripts`, and into e.g. `/admin` or similar?

Reply:
I think the `uploader` module has to remain inside the `recording_uploader` module. If you are talking about the `recording_uploader` module, we could, but I think `scripts` is also a good enough place for it to be in. Let me know

Reply:
Yes I meant to move `recording_uploader` into a new directory, `/admin` or similar.